DevIncubator_Autopark/AllCollections/AutoRepairShop.cs

```diff
@@ -0,0 +1,58 @@
+﻿using DevIncubator_Autopark.Files;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace DevIncubator_Autopark.AllCollections
+{
+    public class AutoRepairShop
+    {
+        private readonly string DirectoryPath = @$"{Directory.GetCurrentDirectory()}\Files\";
+        private readonly Dictionary<string, int> Orders = new Dictionary<string, int>();
+
+        public string FileName { get; }
+        private List<string> _orders;
+
+        public AutoRepairShop(string fileName)
+        {
+            FileName = fileName;
+
+            GetListOrders();
+            CreateDictionary();
+        }
+
+        private void GetListOrders()
+        {
+            var csvHelper = new CsvHelper(@$"{DirectoryPath}{FileName}");
```

Comment:
If you would like to concat some parts of path better to use Path.Combine

Reply:
fixed